modules/mobile/spec/models/appointment_spec.rb

```diff
@@ -3,17 +3,27 @@
 require 'rails_helper'
 
 RSpec.describe Mobile::V0::Appointment, type: :model do
-  describe '.toggle_non_prod_id!' do
-    it 'toggles mocked ids to real ones' do
-      expect(Mobile::V0::Appointment.toggle_non_prod_id!('983')).to eq('442')
+  describe '.convert_non_prod_id!' do
+    it 'converts mocked ids to real ones' do
+      expect(Mobile::V0::Appointment.convert_non_prod_id!('983')).to eq('442')
     end
 
-    it 'toggles real ids back to mocked ones' do
-      expect(Mobile::V0::Appointment.toggle_non_prod_id!('442')).to eq('983')
+    it 'does not change id when already valid' do
+      expect(Mobile::V0::Appointment.convert_non_prod_id!('442')).to eq('442')
     end
 
     it 'keeps secondary identifiers' do
-      expect(Mobile::V0::Appointment.toggle_non_prod_id!('983GC')).to eq('442GC')
+      expect(Mobile::V0::Appointment.convert_non_prod_id!('983GC')).to eq('442GC')
+    end
+
+    context 'when not in production' do
+      before { Settings.hostname = 'api.va.gov' }
+
+      after { Settings.hostname = 'www.example.com' }
+
+      it 'only converts mocked ids' do
```

Comment:
Is this an accurate description? I would expect it to be something like, "does not convert mocked ids". I'm not sure why they're being called mocked, but I think you have a better handle on the underlying issue than I do.

Reply:
Oops, that's what I get for trying to work before having coffee.

Reply:
Never work before coffee.